crates/oxc_prettier/src/format/mod.rs

```diff
@@ -2125,7 +2125,11 @@ impl<'a> Format<'a> for PropertyKey<'a> {
                     if need_quote {
                         dynamic_text!(
                             p,
-                            string::print_string(p, literal.raw, p.options.single_quote)
+                            string::print_string(
+                                p,
+                                literal.raw.as_ref().unwrap(),
```

Comment:
It's possible that prettier will be run on an AST which has been through transformer, so may contain generated nodes, which may have `raw: None`. So need to fall back to converting `literal.value` to a string if `raw.is_none()`.